src/SkeletalTrapezoidation.h

```diff
@@ -22,6 +22,7 @@
 
 namespace arachne
 {
+    using namespace cura;
 
 extern bool generate_MAT_STL;
```

Comment:
Isn't this also debug code?

Reply:
No, this is needed for compiling on windows. Unless we want to change every line that calls into the cura namespace from `function(...` to `cura::function`,

Reply:
I suppose eventually we'll want to remove the `arachne` namespace altogether, to keep things simple?

For now it's necessary though because a lot of classes and functions are literally copy-pasted from the main CuraEngine code and so they have the same names.